### ISSUE TITLE
ActiveRecord::RecordInvalid: Validation failed: From domain doesn't have a DNS record configured correctly for cuttlefish._domainkey.foo.com

### ISSUE THREAD

**Issue by jseparovic** [mlandauer/cuttlefish#340]
Using the docker-compose instructions from the Readme I run into:

```
ActiveRecord::RecordInvalid: Validation failed: From domain doesn't have a DNS record configured correctly for cuttlefish._domainkey.foo.com
```

I't looks like the Readme is missing some install instructions

**Comment by s01ipsist** [comment]
This happens during the `rake db:seed` call which is part of `rake db:setup`.

```
acting_app = smart_team.apps.create!(
  name: "Acting Twins",
  legacy_dkim_selector: true,
  dkim_enabled: true,
  from_domain: "foo.com"
)
```
fails because it tries to validate that dkim record actually exists.
You can work around it by turning off `dkim_enabled`, eg.
```
acting_app = smart_team.apps.create!(
  name: "Acting Twins",
  legacy_dkim_selector: true,
  dkim_enabled: false,
  from_domain: "foo.com"
)
```

**Comment by fazlerabbi37** [comment]
I can confirm that editing `db/seeds.rb` to switch `dkim_enabled` to `false`  solves the problem.

**Comment by mlandauer** [comment]
Sorry all for this. This has now been fixed in 8daaf1bc341cb65e11e734ee348454f3c68f4ecc
